src/main/resources/configuration/Configuration.properties

```diff
@@ -1,7 +1,9 @@
-url=http://demowebshop.tricentis.net/
+url=http://demowebshop.tricentis.com/
 chromeDriverPath=
 geckoDriverPath=
 userName=soa_user
 password=Soa_user5!
 browser=chrome
-emptyParameter=
\ No newline at end of file
+emptyParameter=
+waitTimeout=15
```

Comment:
don't forgot to change here either, if update in method to milliseconds

Reply:
done